contracts/.solhint.json

```diff
@@ -3,11 +3,10 @@
   "plugins": ["prettier"],
   "rules": {
     "code-complexity": ["error", 7],
```

Comment:
On Arbitrum, Setup worked w/o issue, but:

* The toast took 15+ seconds to change whereas the tx was shown as confirmed almost instantly on arbiscan
* After it was confirmed, in addition to seeing the toast saying it succeeded, I saw this error:

<img width="765" alt="Screen Shot 2022-03-17 at 11 22 47 PM" src="https://user-images.githubusercontent.com/1479802/158931811-03b1eb43-d3a8-4cea-b30e-b580870ba532.png">

The send did succeed, heres the tx: https://arbiscan.io//tx/0xbd34fa2b4fdf38cf8a27c677a7d11097b965c535ab03c0fbb63a974dcfaeee2e

Reply:
On the withdraw side, the fee estimation looks good:

<img width="356" alt="Screen Shot 2022-03-17 at 11 27 27 PM" src="https://user-images.githubusercontent.com/1479802/158932253-0751eee4-7408-4de4-80d0-0bffe504c26d.png">

However clicking the button to edit it yields the precision related error:

<img width="768" alt="Screen Shot 2022-03-17 at 11 25 46 PM" src="https://user-images.githubusercontent.com/1479802/158932295-f72ba761-a0c5-4e50-a3d3-1f407f0dd8d9.png">

Reply:
Yeah, I was going to say, the rounding on that fee estimate did not look correct. Seems like this is a bug

Reply:
That same `toPrecision() argument must be between 1 and 100` error shows up when clicking the edit button on Optimism also

Reply:
> The toast took 15+ seconds to change whereas the tx was shown as confirmed almost instantly on arbiscan

This seems to happen sometimes on Optimism too. I'd guess the issue is either that the default provider config [we use](https://github.com/ScopeLift/umbra-protocol/blob/a71af1425c497c046a960ca0dea4e6fc31a51d7e/frontend/src/utils/alerts.ts#L71) to monitor tx status may not be polling frequently enough, or on L2s it waits for additional confirmations, but need to look more into it

Reply:
> On Arbitrum, Setup worked w/o issue, but:
> - After it was confirmed, in addition to seeing the toast saying it succeeded, I saw this error:

I was not able to reproduce this. @apbendi if you know how let me know. Worth noting that I have metamask configured to use Alchemy RPC's over default ones because it's less flaky and more reliable

Reply:
* toPrecision error is gone
* Not seeing the unsupported block number error anymore, at least in today's testing 🤷‍♂️ 
* Understood on the toast issue— we can tackle that separately I think